cfpb/consumerfinance.gov
# Search: More accessible descriptions for search buttons

## Changes

- Search: More accessible descriptions for search buttons


## How to test this PR

1. Pull branch and visit http://localhost:8000/es/obtener-respuestas/que-debo-hacer-cuando-un-cobrador-de-deudas-se-comunica-conmigo-es-1695/ and compare to https://consumerfinance.gov/es/obtener-respuestas/que-debo-hacer-cuando-un-cobrador-de-deudas-se-comunica-conmigo-es-1695/
Check the global search and ask CFPB search buttons aria label. It should have a more descriptive label than just "Search"
Do the same for:
 - http://localhost:8000/rules-policy/regulations/search-regulations/results/?regs=1041
 - http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/

You can also use the WebAIM extension to see the labels without inspecting the page source.
